### PR TITLE
Add AKS document

### PR DESCRIPTION
**Pull request by SparkSnail**
1.Add kubeflow in experiment config document
2.Add AKS in kubeflow document

### CODE REVIEW

File: docs/ExperimentConfig.md [hunk 1]
```diff
@@ -141,12 +140,17 @@ machineList:
 * __trainingServicePlatform__
   * Description
       
-	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__}.  
-	* __local__ mode means you run an experiment in your local linux machine.  
+	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__, __pai__, __kubeflow__}.  
 	
-	* __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
+    * __local__ mode means you run an experiment in your local linux machine.
```
Comment (yds05) [inline docs/ExperimentConfig.md]: __local__ mode : run an experiment on local ubuntu machine.

File: docs/ExperimentConfig.md [hunk 2]
```diff
@@ -141,12 +140,17 @@ machineList:
...
       
-	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__}.  
-	* __local__ mode means you run an experiment in your local linux machine.  
+	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__, __pai__, __kubeflow__}.  
 	
-	* __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
+    * __local__ mode means you run an experiment in your local linux machine.  
+	
+	
+    * __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.
```
Comment (yds05) [inline docs/ExperimentConfig.md]: __remote__ mode: submit trial jobs to remote ubuntu machines, and __machineList__ field should be filed in order to set up SSH connection to remote machine

File: docs/ExperimentConfig.md [hunk 3]
```diff
@@ -141,12 +140,17 @@ machineList:
...
 	
-	* __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
+    * __local__ mode means you run an experiment in your local linux machine.  
+	
+	
+    * __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
 
-	* __pai__ mode means you submit trial jobs to [OpenPai](https://github.com/Microsoft/pai) of Microsoft. For more details of pai configuration, please reference [PAIMOdeDoc](./PAIMode.md)
+	
+    * __pai__ mode means you submit trial jobs to [OpenPai](https://github.com/Microsoft/pai) of Microsoft. For more details of pai configuration, please reference [PAIMOdeDoc](./PAIMode.md)
```
Comment (yds05) [inline docs/ExperimentConfig.md]: The same way to simplify

File: docs/ExperimentConfig.md [hunk 4]
```diff
@@ -141,12 +140,17 @@ machineList:
...
+   
+    * __kubeflow__ mode means you submit trial jobs to [kubeflow](https://www.kubeflow.org/docs/about/kubeflow/), nni support kubeflow based on normal kubernets and [azure kubernets](https://azure.microsoft.com/en-us/services/kubernetes-service/).
```
Comment (yds05) [inline docs/ExperimentConfig.md]: The same way to simplify

File: docs/ExperimentConfig.md [hunk 5]
```diff
@@ -160,6 +164,14 @@ machineList:
     __useAnnotation__ means whether you use annotation to analysis your code and generate search space. 
 	   
 	    Note: if you set useAnnotation=True, you should not set searchSpacePath.
+
+* __nniManagerIp__
+  * Description
+   
+    __nniManagerIp__ set the IP address of your machine which the nni manager process runs on. This field is an optional choice, if you don't set nniManagerIp, nni will use the IP of etho device.
```
Comment (yds05) [inline docs/ExperimentConfig.md]: set the IP address of the machine on which nni manager process runs. This field is optional, and if it's not set, eth0 device IP will be used instead.

File: docs/ExperimentConfig.md [hunk 6]
```diff
@@ -160,6 +164,14 @@ machineList:
...
+
+        Note: if you don't have eth0 device in your machine, we suggest you to set nniManagerIp manually.
```
Comment (yds05) [inline docs/ExperimentConfig.md]: run ``ifconfig`` on NNI manager's machine to check if eth0 device exists. If not, we recommend to set nnimanagerIp explicitly.

File: docs/ExperimentConfig.md (hunk 1 above)
Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to new format

File: docs/ExperimentConfig.md (hunk 2 above)
Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to new format

File: docs/ExperimentConfig.md (hunk 4 above)
Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to new format

File: docs/ExperimentConfig.md (hunk 5 above)
Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to new format

File: docs/ExperimentConfig.md (hunk 6 above)
Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to new format.

**Pull request merged**